packages/framework/src/Form/Constraints/UniqueBillingAddress.php

```diff
@@ -21,7 +21,7 @@ class UniqueBillingAddress extends Constraint
      * @var array<string, string>
      * @phpcsSuppress SlevomatCodingStandard.TypeHints.PropertyTypeHint.MissingNativeTypeHint
      */
-    protected static $errorNames = [
+    protected const array ERROR_NAMES = [
```

Comment:
I see this change in +-2 classes only, what about the rest of the classes that extend `Symfony\Component\Validator\Constraint`?